package/README.md

```diff
@@ -0,0 +1,59 @@
+# Short Term History (STH) package
+
+This directory contains tools to make and distribute STH in RPM format used, mainly, in Red Hat, CentOS and
+Fedora Linux distributions.
+
+With the RPM is included:
+- STH SW
+- node modules that are needed by STH
+- [sth](rpm/SOURCES/etc/init.d/sth "sth") init.d script with standard operations
+- Daily log rotation [logrotate-sth-daily](rpm/SOURCES/etc/logrotate.d/logrotate-sth-daily "logrotate")
+- Creation of directories: `/var/log/sth` and `/var/run/sth`
+- User `sth` is created and is the propietary of the installed files and running STH process
+
+## Package STH SW
+
+**Prerequisites:** To make RPM is need to have rpm build tools (rpmbuild executable), NodeJS and 
```

Comment:
I've seen this was fixed in https://github.com/telefonicaid/IoT-STH/commit/1b710efe40b04c6944ce7a18de42fa3ebcc8807d